.github/workflows/test-deployer.yaml

```
@@ -0,0 +1,30 @@
+name: Run the deployer's tests
+
+on:
+  push:
+    paths:
+      - "**/deployer/**"
+      - "tests/**"
+  pull_request:
+    paths:
+      - "**/deployer/**"
+      - "tests/**"
```

I think `workflow_dispatch` to allow for a manual invocation of the workflow is great to have. I've been around the jupyterhub org and added that to many test suites. I suggest adding that by default unless you know it doesn't make sense to re-run from the default branch.

Tests is really helpful to re-run though. Often tests can start failing in a PR and you can then go to the default branch and invoke a test workflow to see if it has started failing also there.

```suggestion
      - "tests/**"
  workflow_dispatch:
```

---

Yeah, this is just me testing - it's not supposed to be perfect